### ISSUE TITLE
Port this integration to HACS

### ISSUE THREAD

**Issue by DaviPtrs** [DaviPtrs/hass-unified-remote#6]
https://hacs.xyz/docs/publish/start

**Comment by jsp196** [comment]
Any chance this can happen to make installation easier?  I'm a new HA user and am not sure how to install manually.

**Comment by DaviPtrs** [comment]
Absolutely, I just need to get some time, or some volunteer to take this issue...

**Comment by amathews84** [comment]
Hello I was wondering if you ever made this available in HACS?

**Comment by daniilink** [comment]
Add it to HACS pls I struggled 2 days spending via trying add this im newbie
